language_detector/main.py

```diff
@@ -6,6 +6,16 @@
 
 
 def detect_language(text, languages=LANGUAGES):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    stats = {}                          #Creating a dynamic counter
+    for language in languages:
+        name = language['name']
+        stats[name] = 0
+    
+    for item in text.split(' '):        #iterates through each word in text
+        for language in languages:      #iterates through each language
+            name = language['name']
+            for word in language.get('common_words'): #iterates through each common word
```

Comment:
a little less effeicent to iterate through the words in both the text and language dictionary but it is more accurate as your get less false matches than using `for item in language['common_words']`